react/src/components/RegularSeasonTable.js

```diff
@@ -85,19 +86,18 @@ const RegularSeasonTable = ({ column, data, isLoaded, direction, handleSort }) =
               data,
               (
                 {
+                  isChamp,
                   leagueID,
                   leaguename,
-                  teamID,
-                  teamname,
                   FFname,
                   Wins,
                   Losses,
                   pointsFor,
                   pointsAgainst,
                   coachRating,
-                  isChamp,
```

Comment:
Actually, I think you may just need a pull here? I don't think you need to change anything in this data mapping.

Reply:
NVM I see it's just reorganizing 3 vars and removing an unused one. If order won't matter here I can just merge the PR.